elfpy/data/postgres.py

```diff
@@ -475,9 +553,9 @@ def get_agents(session: Session, start_block: int | None = None, end_block: int
     query = session.query(WalletInfo.walletAddress)
     # Support for negative indices
     if (start_block is not None) and (start_block < 0):
-        start_block = _get_latest_block_number_wallet_info(session) + start_block + 1
+        start_block = get_latest_block_number_from_table(WalletInfo.__tablename__, session) + start_block + 1
     if (end_block is not None) and (end_block < 0):
-        end_block = _get_latest_block_number_wallet_info(session) + end_block + 1
+        end_block = get_latest_block_number_from_table(WalletInfo.__tablename__, session) + end_block + 1
```

Comment:
Why did you change these from private functions?

Reply:
Ok, that makes sense. I recommend that you keep it private until this issue actually comes up. In other words, go ahead and hand-roll a new access function for the next few new tables. Once we have a collection of them we can discuss generalizing.